packages/remark-parse/lib/parser.js

```diff
@@ -5,17 +5,14 @@
  * @module remark:parse
  * @fileoverview Markdown parser.
  */
+import xtend from 'xtend';
```

Comment:
`xtend` is very small and simple, and used a lot throughout all my plugins, so it probably doesn’t cost anything to keep it in, whereas I don’t use `lodash.merge` so it’ll probably add to the bundled size!